src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,17 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+
+  max-width: 1024px;
+  margin: 0 auto;
+  padding: 50px 40px;
+
+  &__card {
+    margin-right: 48px;
+    margin-bottom: 46px;
```

Comment:
what about left out margins on the bottom? remove them
![image](https://user-images.githubusercontent.com/37161315/215912554-08fa58c7-918c-4bef-8bba-c4a2a4d02e6f.png)